CONTRIBUTING.md

```diff
@@ -1,6 +1,24 @@
 # C#/WinRT Contributor's Guide
 
-Below is our guidance for how to report issues, propose new features, and submit contributions via Pull Requests (PRs). Please refer to [building and running C#/WinRT](./README.md#Building-and-running-cwinrt) to get started on building the repo.
+Below is our guidance for how to build the repo, report issues, propose new features, and submit contributions via Pull Requests (PRs). 
+
+## Building the C#/WinRT repo
+
+C#/WinRT currently requires the following packages, or newer, to build:
+
+- [Visual Studio 16.8](https://visualstudio.microsoft.com/downloads/) 
+- [.NET 5.0 SDK](https://dotnet.microsoft.com/download/dotnet/5.0) 
+- [.NET Core 2.1 SDK](https://dotnet.microsoft.com/download/dotnet-core/2.1)
+- [nuget.exe 5.8.0-preview.3](https://www.nuget.org/downloads)
+- Microsoft.WinUI 3.0.0-preview1.200515.3
+
+The `build.cmd` script takes care of all related configuration steps and is the simplest way to get started building C#/WinRT. It installs prerequisites such as nuget and the .NET 5 SDK, configures the environment to use .NET 5 (creating a `global.json` if necessary), builds the compiler, and builds and executes the unit tests. To build C#/WinRT, follow these steps: 

```

Comment:
should we add details on configuration - x64/x86, debug/release?

Reply:
Would be good imo; alternatively we update build.cmd to give a helpful output if you pass it an option like -h or --help

Reply:
This made me also think that there is a relevant detail for x86 -- https://github.com/microsoft/CsWinRT/issues/341